### DISCUSSION TITLE
More git tags for one version bumping commit

### DISCUSSION THREAD

**Discussion by csaszika** [jscutlery/semver#689] [General]
So what we currently have: (commits timeline is going down)

```
feat(*): some api and components are created
chore(api): release version 0.0.1
chore(components): release version 0.0.1
```
which put git tags on the last 2 commits like this:
```
api-0.0.1
components-0.0.1
```

Is it possible currently to create this commit history?:

```
feat(*): some api and components are created
chore(*): release api version 0.0.1, components version 0.0.1
```
which means we will put more git tags on the same commit (in this case api-0.0.1 and components-0.0.1).

I know if we have many libraries it can be a long commit message, but probably it can be solved in a way.
For example, allowing longer commit messages or overriding default release commit message in "root config".

just an example:
```
chore(*): release api, components...
```
For me the commit message is less important. If it does not contain "version" and version numbers, idc. I'm using git tags for deploying and other stuffs. (It is really personal point of view).

What do you think about it?

**Comment by edbzn** [comment]
Hi, with the actual design it's almost impossible so this is something we would like to support in Semver 3 #655.

**Reply by csaszika** [reply]
oh thanks, I did not noticed, sounds cool
